app/api/utils.py

```diff
@@ -390,6 +390,30 @@ def parse(self, file):
         if data:
             yield data
 
+class JSONParser(FileParser):
```

Comment:
JSONParser is not enough to parse JSON file. For example:

```
[{"text": "Terrible customer service.", "labels": ["negative"]},{"text": "Really great transaction.", "labels": ["positive"]},{"text": "Great price.", "labels": ["\
 positive"]}]
```

Reply:
this should be fixed now since I replaced it with a RegEx that finds each valid json line. I tried multiple special cases and all have worked so far. If you still find a case that is not working, plesase let me know